### PR TITLE
Add warning about effect of ICE policy. Fixes #214

### LINKED ISSUE #214: Candiates in use on Policy Change
If a candidate is in use and Policy changes such that this candidate is not longer allowed, do we have text that says to stop using that candidate and restart ice?

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

**Comment by fluffy** [comment]
LGTM

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -1197,7 +1197,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             gathering phase. If the policy is being made more liberal (i.e.,
             more permitted candidates), the 'needs-ice-restart' bit will be set.
             If the policy is being made more strict, no ICE restart is needed,
-            and the bit will not be set. Either way, changes to the policy have
+            and the bit will not be set. Any candidates which have already been
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Do you want to add a clause explaining the rationale here?

e.g. "as any relevant information has already been revealed"

**Comment by juberti** [comment]
lgtm with one comment

**Comment by fluffy** [comment]
LGTM

**Comment by juberti** [comment]
I think this is obsoleted by #225

**Pull request closed without merging**